Link private member documentation to weakmaps/closures (#17761)
mdn/content: files/en-us/web/javascript/reference/global_objects/weakmap/index.md

}
```

### Emulating private members

> Weakmaps allow one to associate private data to an object. Using a weakmap has the following benefits:

Reviewer: ```suggestion
Developers can use a {{jsxref("WeakMap")}} to associate private data to an object, with the following benefits:
```
